### ISSUE TITLE
tester.compile doesn't recompile VCS

### ISSUE THREAD

**Issue by THofstee** [leonardt/fault#127]
The SystemVerilog targets actually don't compile the Verilog testbench unless you call `compile_and_run` instead of just `compile`.

**Comment by THofstee** [comment]
Notably, I just want to recompile the VCS testbench I'm using and run `./simv` manually myself.

**Comment by leonardt** [comment]
Makes sense, I think this feature (compile) was added for verilator but never ported for the SV targets
